web/src/queries/software.ts

```diff
@@ -113,14 +114,28 @@ const useConfigMutation = () => {
   return useMutation(query);
 };
 
+type QueryHookOptions = {
+  suspense: boolean;
+};
+
```

Comment:
This is already declared in _types/queries.ts_. I guess it is a leftover and would be sufficient importing it.